modules/DMZ/data_kit/process_folder.py

```diff
@@ -0,0 +1,45 @@
+# Import needed libraries
+from PIL import Image  # image libraries and operations
+import os, sys  # directory libraries
+import imghdr  # image file recognition
+import numpy  # numpy arrays
+
+# Sample Paths
+# /mnt/c/Users/Owner/Pictures/SamplePictures/
+# /mnt/c/Users/Tayo/Documents/SE2CS4365/TestImages/
+
+# list of image types
+imgfiles = ['rgb', 'gif', 'pbm', 'pgm', 'ppm', 'tiff', 'rast', 'xbm', 'jpeg', 'jpg', 'bmp', 'png', 'webp', 'exr']
+
+
+# Splitting up this module into smaller ones
+
+# Given a filepath and size, resizeFolder will enter the folder
+# and edit the files to be square, with lxw as sizexsize
+def resizeFolder(path, size):
+    dirs = os.listdir(path)  # store the path.
+
+    for item in dirs:
+        if imghdr.what(path + item) in imgfiles:  # if the file has the correct type
+            img = Image.open(path + item)  # .convert('L') #open and convert to greyscale
+            a, b = os.path.splitext(path + item)
+            newImg = img.resize((size, size), Image.ANTIALIAS)  # resize and keep quality
+            newImg.save(a + 'edit.jpg', 'JPEG', quality=90)  # save the image with edit appended to it
+
+
+# given a directory string, will return the numpyarray of square images with lxw = size x size
+def getImageArray(path, size):
```

Comment:
According to [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html#Naming) method name should be `get_image_array` or something similiar